templates/master/00-master/_base/files/usr-local-bin-openshift-recovery-tools-sh.yaml

```diff
@@ -30,15 +30,13 @@ contents:
            exit 1
         fi
       else
-        GOOGLE_URL=https://storage.googleapis.com/etcd
-        DOWNLOAD_URL=${GOOGLE_URL}
-
-        echo "Downloading etcdctl binary.."
-        curl -s -L ${DOWNLOAD_URL}/${ETCD_VERSION}/etcd-${ETCD_VERSION}-linux-amd64.tar.gz -o $ASSET_DIR/tmp/etcd-${ETCD_VERSION}-linux-amd64.tar.gz \
-          && tar -xzf $ASSET_DIR/tmp/etcd-${ETCD_VERSION}-linux-amd64.tar.gz -C $ASSET_DIR/shared --strip-components=1 \
-          && mv $ASSET_DIR/shared/etcdctl $ASSET_DIR/bin \
-          && rm $ASSET_DIR/shared/etcd \
-          && $ETCDCTL_BIN version
+        local etcdimg="{{.Images.etcdKey}}"
+        local etcdctr=$(podman create ${etcdimg})
+        local etcdmnt=$(podman mount "${etcdctr}")
+        cp ${etcdmnt}/bin/etcdctl $ASSET_DIR/bin
```

Comment:
you tested this? pretty sure the path is `/usr/bin/etcdctl`[1]

[1] https://github.com/openshift/etcd/blob/e30a18f29c8bf1f16507d7983dde2aa51962d6c7/Dockerfile.rhel#L15

Reply:
yes i did test this - and this worked.

Reply:
confirmed in manual testing.

Reply:
> lrwxrwxrwx. 1 root root   7 Sep  2 12:45 bin -> usr/bin

Reply:
ah..yep :)

Reply:
I tested it and it works too.